test/langtools/tools/javac/ConditionalExpressionResolvePending.java

```diff
@@ -34,7 +34,7 @@
  *      jdk.compiler/com.sun.tools.javac.util
  * @build toolbox.ToolBox toolbox.JavacTask
  * @build combo.ComboTestHelper
- * @compile --enable-preview -source ${jdk.version} ConditionalExpressionResolvePending.java
+ * @compile ConditionalExpressionResolvePending.java
  * @run main/othervm --enable-preview ConditionalExpressionResolvePending
```

Comment:
do you still need the @run main/othervm?

Reply:
Yes, that's still needed. The test internally compiles some code with --enable-preview and then tries to load and run the code in a separate ClassLoader in the same VM. So --enable-preview at runtime is required.

Reply:
ok looks good